packages/round-manager/src/features/api/services/program.ts

```diff
@@ -78,66 +78,42 @@ export const programApi = api.injectEndpoints({
         try {
           account = ethers.utils.getAddress(account)
 
-          const programFactory = new ethers.Contract(
-            programFactoryContract.address!,
-            programFactoryContract.abi,
-            global.web3Provider
+          const res = await graphql_fetch(
```

Comment:
would it be worth pushing all the queries into it's own folders for the sake of re-usability? not sure how meaningful it would be

Reply:
i thought about this but figured the queries wouldn't be reused elsewhere since components only interact with the services hence i only abstracted the `graphql_fetch` core functionality